pom.xml

```diff
@@ -30,6 +30,7 @@
         <ossindex.version>3.2.0</ossindex.version>
         <jgit.version>6.8.0.202311291450-r</jgit.version>
         <maven.surefire-failsafe.version>3.2.5</maven.surefire-failsafe.version>
+        <org.mapstruct.version>1.5.5.Final</org.mapstruct.version>
```

Comment:
`mapstruct.version` is enough, since we don't use the FQN in the other properties.